packages/bsky/src/auth-verifier.ts

```diff
@@ -159,19 +167,19 @@ export class AuthVerifier {
     }
   }
 
-  adminService = async (reqCtx: ReqCtx): Promise<AdminServiceOutput> => {
+  modService = async (reqCtx: ReqCtx): Promise<ModServiceOutput> => {
     const { iss, aud } = await this.verifyServiceJwt(reqCtx, {
       aud: this.ownDid,
-      iss: [this.adminDid],
+      iss: [this.modServiceDid, `${this.modServiceDid}#atproto_mod`],
```

Comment:
I think `#atproto_mod` is the id of the key.  Do we want to use the id of the service here, map it to the id of the key when verifying?

Reply:
Yup we chatted about this - I changed this to be the serviceId to keep it inline with possible future updates 👍